src/screens/Asl/components/AslTable/index.js

```diff
@@ -61,19 +61,21 @@ const AslTable = props => {
         width: 400,
         height: 200
     })
-    const oriPos = useRef({
+    const [oriPos, setOriPos] = useState({
         // player position
         top: 0,
         left: 0,
         // cursor position
         cX: 0, 
         cY: 0
     })
-    const isDown = useRef(false);
-    const direction = useRef();
+    const [isDown, setIsDown] = useState(false);
+    const [direction, setDirection] = useState('');
+
 
     const apiInstance = axios.create({
-      baseURL: 'https://ct-dev.ncsa.illinois.edu',
+      // baseURL: 'https://ct-dev.ncsa.illinois.edu',
+      baseURL: window.location.hostname,
```

Comment:
Sorry, no this assumes that the frontend is running on the same host as the server. There's a config variable that holds the server.
Please use the the CT http request and api classes (that wraps axios)
https://github.com/classtranscribe/FrontEnd/blob/staging/src/utils/cthttp/index.js

Reply:
e.g. api usage here-
https://github.com/classtranscribe/FrontEnd/blob/staging/src/screens/Course/model.js
[Is imported from utils, so lets open utils/index.js](https://github.com/classtranscribe/FrontEnd/blob/staging/src/utils/index.js)
which has 
export { api } from './cthttp';
So let's look at that-
https://github.com/classtranscribe/FrontEnd/blob/staging/src/utils/cthttp/index.js
Oohh- so the the right way to do this is to define an Entity for each Glossary item
(but you could probably drop down to cthttp request if you really wanted to.)